.dockerignore

```diff
@@ -0,0 +1,15 @@
+.dockerignore
```

Comment:
Maybe we could have something like:
```Dockerfile
*
!src
!package*.json
```
instead?

Reply:
What do you think about it?

Reply:
I'm not very familiar with docker images, this change prevents the source files to go out to production?

Reply:
`.dockerignore` prevent the docker layer cache to be invalidated (when you change a trivial file like a `README.md` file for example). I suggest we ignore everything by default and then whitelist the stuff that should bust the cache when they're modified.